go.mod

```diff
@@ -82,7 +82,7 @@ require (
 	k8c.io/operating-system-manager v0.4.0
 	k8s.io/api v0.23.4
 	k8s.io/apiextensions-apiserver v0.23.4
-	k8s.io/apimachinery v0.23.4
+	k8s.io/apimachinery v0.23.5
```

Comment:
Why update this one dependency?

Reply:
no specific reason, i will change it back to what it was.